GOVERNANCE.md

```diff
@@ -9,9 +9,21 @@ The [nodejs/node](https://github.com/nodejs/node) GitHub repository is
 maintained by Collaborators who are added by the CTC on an ongoing basis.
 
 Individuals identified by the CTC as making significant and valuable
```

Comment:
So all collaborators are approved by the CTC? Assuming collaborators will now roughly map to [`@nodejs/members`](https://github.com/orgs/nodejs/teams/members/members), does this mean everyone who joins the org needs CTC approval? Even if they're joining CommComm for example?

Also is this the current process? If not it seems like something that could use more general `members` feedback.

The answer could totally be "this is a hard question, let's roll with this and improve it going forward".